how-we-work/iteration.qmd

```diff
@@ -31,98 +31,161 @@ research and academic environments don't foster nor support nor are
-    -   Show off and (optionally) demo the output all together as a
-        team. We'll assign someone to be responsible for showing or
-        trying out the demo before the meeting.
-    -   (Optionally done occasionally, but strongly suggested) Include
-        someone external (a potential user) to try out the product/demo
-        and give some feedback.
-    -   Do a "retrospective", where we individually write down how
-        things went ("keep doing" category) and what could be improved
-        or removed ("stop doing/improve on" category) in the process.
-        This should ideally be done in a collaborative
-        "notetaking/post-it notes" system
-        ([Figma](https://www.figma.com/) is really nice for this).
\ No newline at end of file
+Based on these inspirations, an iteration for us follows a general
+sequence of steps.
+
+## Future iteration planning
+
+Team lead sets everything up for the next two or three iterations.
+
+-   Will create an iteration within the main "Team project planning"
+    board (which are all listed
+    [here](https://github.com/orgs/seedcase-project/projects/), which is
+    found in the Settings in the board.
+-   Will write the title of the iteration as the goal for it, while
+    keeping the goal focused, small, achievable, and considerate of the
+    time we have available.
+-   Will add the main tasks to achieve the iteration goal to the board.
+-   Will plan iterations so that they either end on a major holiday or
+    are right in the middle of it. If possible, don't plan the start of
+    iterations a few days before major holidays start.
+-   Will plan and decide on how long the future iterations should be
+    (can range from 2-6 weeks, but prefer shorter iterations over longer
+    ones). So far, we find that 3 week iterations work the best.
+
+## Start of iteration
+
+::: callout-important
+We track our iterations' progress and tasks on GitHub with [this project
+board](https://github.com/orgs/seedcase-project/projects/18/).
+:::
+
+An iteration starts with a first planning meeting that is scheduled well
+in advance, for between 1-2 hours and (preferably) right after the
+previous iteration's debrief and retrospective. Preferably this meeting
+is in person. The aim of the meeting to discuss and plan the specifics
+of the iteration.
+
+::: callout-note
+A basic agenda for the meeting might be:
+
+-   Assign a timekeeper.
+-   Review and agree on the iteration aim and end date.
+-   Review the list of tasks already listed.
+-   Brainstorm and add any other issues as needed.
+-   Assign a team member to each task.
+-   Schedule the next iteration's end and start meetings.
+:::
+
+::: panel-tabset
+### Before meeting
+
+In general, only the team lead is required to prepare for the planning
+meeting. Before the planning meeting, the team lead will move any
+existing tasks (GitHub issues) into the iteration that is relevant to
```

Comment:
```suggestion
existing tasks (GitHub Issues) into the iteration that is relevant to
```

@signekb small thing, when issues are used with the word GitHub, they are capitalized, in line with GitHub's own documentation ("GitHub Issues" vs "create a new issue")

Reply:
Ah, you're right!